scripts/zones/Arrapago_Reef/mobs/Medusa.lua

```diff
@@ -6,11 +6,19 @@
 -----------------------------------
 package.loaded["scripts/zones/Arrapago_Reef/TextIDs"] = nil;
 -----------------------------------
-mixins = {require("scripts/mixins/job_special")};
+require("scripts/globals/status");
 require("scripts/zones/Arrapago_Reef/TextIDs");
+require("scripts/zones/Arrapago_Reef/MobIDs");
 require("scripts/globals/titles");
+mixins = {require("scripts/mixins/job_special")};
```

Comment:
pls don't shuffle the exiting mixins line down. personally I like making the widest lines at the top of the same "type" 
so it'd be
```lua
-----------------------------------
package.loaded["scripts/zones/Arrapago_Reef/TextIDs"] = nil;
-----------------------------------
mixins = {require("scripts/mixins/job_special")};
require("scripts/zones/Arrapago_Reef/TextIDs");
require("scripts/zones/Arrapago_Reef/MobIDs");
require("scripts/globals/status");
require("scripts/globals/titles");
```